pkg/master/controller.go

```diff
@@ -177,7 +178,7 @@ func (c *Controller) UpdateKubernetesService(reconcile bool) error {
 			return err
 		}
 		endpointPorts := createEndpointPortSpec(c.PublicServicePort, "https", c.ExtraEndpointPorts)
-		if err := c.EndpointReconciler.ReconcileEndpoints("kubernetes", c.PublicIP, endpointPorts, reconcile); err != nil {
+		if err := c.EndpointReconciler.ReconcileEndpoints("apiservers", "kubernetes", c.PublicIP, endpointPorts, reconcile, time.Now()); err != nil {
```

Comment:
In https://github.com/kubernetes/kubernetes/pull/34005#discussion_r82660381 a different config map was discussed: with substitutes for command line flags.

Config map introduces by this PR keeps TTLs for servers. I think this config map should be treated the same way as kubernetes service (it does not have command line flag).